sm_cnn/trec_dataset.py

```diff
@@ -1,16 +1,14 @@
 from torchtext import data
-import os
 
 class TrecDataset(data.TabularDataset):
     dirname = 'data'
     @classmethod
 
-    def splits(cls, question_id, question_field, answer_field, external_field, label_field,
-               train='train.tsv', validation='dev.tsv', test='test.tsv'):
+    def splits(cls, question_id, question_field, answer_field, external_field, label_field, root='.data',
+               train='trecqa.train.tsv', validation='trecqa.dev.tsv', test='trecqa.test.tsv'):
```

Comment:
Seems like you updated to torchtext 0.2.0? If so, please remove instructions to check out a particular torchtext commit from the README and put torchtext 0.2.0 in `requirements.txt`.